modules/running-your-dynamic-plugin.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment.
+
+
+. Run Bridge locally and instruct it to proxy requests directly to your local plugin asset server (web server hosting the plugin's generated assets): `./bin/bridge -plugins console-demo-plugin=http://localhost:9001/`
+
+. Inspect the value of `window.SERVER_FLAGS.consolePlugins` to see the list of plugins which the console loads upon its startup.
```

Comment:
I think it is better to check what `bridge` logs on start up to see that their plugin is enabled.